libs/saksbehandling-common/src/main/kotlin/feilhaandtering/Preconditions.kt

```diff
@@ -0,0 +1,63 @@
+package no.nav.etterlatte.libs.common.feilhaandtering
```

Comment:
Jeg tar det gjerne på engelsk jeg egentlig, men liker dokumentasjonen her.
Skal det først være på norsk burde vel filnavnet her være på norsk og?

Reply:
Hvis du har et godt alternativ til `Preconditions` er det bare å skrike ut! Jeg kalte det `Preconditions` for å tydeliggjøre at det skal erstatte `kotlin.Preconditions`, men er åpen for alternativer. 

Jeg synes direkte oversatte funksjonsnavn gjør det lettere å "kopiere" formatet uten å ha masse rare pre-/suffiks på alt. Hvis vi skal ha det på engelsk burde vi i så fall bli enige om et mønster som gir mening, uten at vi ender opp med navn som `checkNotNullKastUgyldigForespoerselException`